setup.py

```diff
@@ -2,9 +2,9 @@
 
 setup(
     name='django-yaml-redirects',
-    version='0.4',
-    author='Robin Winslow',
-    author_email='robin@canonical.com',
+    version='0.5',
+    author='Canonical Webteam',
+    author_email='webteam@canonical.com',
```

Comment:
not sure about this - if someone wants to complain about this package, do we need our whole team to be spammed?